SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift

```diff
@@ -17,15 +40,19 @@ public class SYMKBrowseMapViewController: UIViewController {
         if let view = (view as? SYMKBrowserMapView) {
             view.compass.viewModel = SYUICompassViewModel(course: 0, autoHide: true)
             view.compass.isHidden = !useCompass
+            view.recenter.setup(with: ActionButtonViewModel(title: "", icon: SygicIcon.positionLockIos, style: .secondary))
+            view.recenter.isHidden = !useRecenterButton
```

Comment:
Tu ma teraz napadlo to napisat trosku krajsie ako som to pripravil..
Najprv si vytvorit `let browseView = SYMKBrowserMapView`
nastavit mu postupne tieto properties a potom uz len zavolat `view = browserView`